WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -602,6 +602,7 @@ def for_senior
         :external_website,
         :use_wca_registration,
         :external_registration_page,
+        :is_using_wca_live_for_scoretaking,
```

Comment:
Very long name. What about `use_wca_live_scoretaking` (analogous to `use_wca_registration` above)?